app/views/shared/_footer.html.erb

```diff
@@ -1,12 +1,12 @@
 <footer class="navbar navbar-inverse site-footer">
   <div class="container-fluid">
     <div class="row">
-      <div class="col-md-offset-4 col-md-6 col-sm-8 white-text">
-        <%= link_to "About Oregon Digital", Hyrax::Engine.routes.url_helpers.help_path, method: :get, class: "white-text right-border" %>
-        <%= link_to "Policies", Hyrax::Engine.routes.url_helpers.help_path, method: :get, class: "white-text right-border" %>
-        <%= link_to "Copyright", Hyrax::Engine.routes.url_helpers.help_path, method: :get, class: "white-text right-border" %>
-        <%= link_to "Contact Us", Hyrax::Engine.routes.url_helpers.help_path, method: :get, class: "white-text right-border" %>
-        <%= link_to "Donate", Hyrax::Engine.routes.url_helpers.help_path, method: :get, class: "white-text" %>
+      <div class="col-md-offset-4 col-md-6 col-sm-8 footer-link-text">
+        <u><%= link_to "About Oregon Digital", Hyrax::Engine.routes.url_helpers.help_path, method: :get, class: "footer-link-text right-border" %></u>
```

Comment:
`<u>` tags have a bit more semantic meaning since HTML 5, and it differs from underline only. These (and all `<a>` tags/links) should be handled with `text-decoration: underline`.
If bootstrap would stop shoving inaccessible links down everyone's throats we wouldn't have this issue!